docker/gcp_pudl_etl.sh

```diff
@@ -225,6 +257,9 @@ if [[ $ETL_SUCCESS == 0 ]]; then
     # should be moved to the triggering github action. Having it here feels fragmented.
     # Distribute outputs if branch is main or the build was triggered by tag push
     if [[ "$GITHUB_ACTION_TRIGGER" == "push" || "$BUILD_REF" == "main" ]]; then
+        # Distribute Parquet outputs to a private bucket
+        distribute_parquet 2>&1 | tee -a "$LOGFILE"
+        DISTRIBUTE_PARQUET=${PIPESTATUS[0]}
```

Comment:
This has to happen before we cleanup the outputs, because we remove all the `parquet/*.parquet` files there.